catalogd/hack/scripts/demo-script.sh

```diff
@@ -12,7 +12,7 @@ kubectl cluster-info --context kind-kind
 sleep 10
 
 # use the install script from the latest github release
-curl -L -s https://github.com/operator-framework/catalogd/releases/latest/download/install.sh | bash
+curl -L -s https://github.com/operator-framework/operator-controller/releases/latest/download/install.sh | bash
```

Comment:
As this change has been made before cutting a latest release, I think this link points to the old `latest` which does not include the monorepo changes. Is the plan to follow-up with cutting a release to make this script work?